prefect_docker/scripts/configuration/settings.toml

```diff
@@ -0,0 +1,30 @@
+[default]
+
+[development]
+debug = true
+
+DATAVERSE_URL="http://dataverse:8080"
```

Comment:
Ik denk data `DATAVERSE_URL` weg kan. Deze setting gebruiken we nergens.
(Staat ook op regel 21.)